inst/shinyApps/opplasting/app.R

```diff
@@ -0,0 +1,239 @@
+library(shiny)
+library(qmongrdata)
+KvalIndData$Aar <- as.integer(KvalIndData$Aar)
```

Comment:
Trengs denne?

Reply:
Det er en enkel måte å sørge for at årstall viser uten desimaler i appen. Det kan selvsagt fikses på andre måter hvis ønskelig.

Reply:
Ville bare ha vekk den j.. lintr-bot-klagingen:)

Skulle vi byttet fra `KvalIndData` til `kvalIndData` og `Aar` til `aar`? Så Så slipper vi klaging fra lintr-bot fremover. Gjelder vel også andre kolonner i dataene...